core/domain/stats_domain_test.py

```diff
@@ -547,6 +720,106 @@ def test_validate(self):
             'Invalid action type: %s' % 'InvalidActionType')):
```

Comment:
Don't forget to remove the old subtests from `test_validate` (lines 694, 700, 707).

Reply:
Edited the issue